src/sink/wavefront.rs

```diff
@@ -335,6 +348,7 @@ mod test {
         assert!(lines.contains(&"test.timer.99 12.101 645181811 source=test-src"));
         assert!(lines.contains(&"test.timer.999 12.101 645181811 source=test-src"));
         assert!(lines.contains(&"test.timer.count 3 645181811 source=test-src"));
+        assert!(lines.contains(&"test.timer.mean 5.434333333333334 645181811 source=test-src"));
```

Comment:
1) Should we test this expected behavior?

2) We should probably call out to the users that certain integer types are not supported.